### DISCUSSION TITLE
Why is Orca Slicer running Edge WebView

### DISCUSSION THREAD

**Discussion by julie777** [OrcaSlicer/OrcaSlicer#7802] [General]
Can someone tell me why Orca Slicer is running a large number of Edge Webview Processes?
I am not sure what it is using Webview for, but 10 processes seems excessive.
![image](https://github.com/user-attachments/assets/eb29bb3f-2bab-45f5-aaf8-39fca3881e0a)

**Comment by DARKFiB3R** [comment]
WebView is used to embed web content (HTML, CSS, and JavaScript) in native applications.

If you have a Bambu printer, this could be MakerWorld stuff, and maybe the video stream in the device tab.

That amount of processes would not be unusual in that case, and it's probably better to have them all separated like that, so if one falls over it doesn't take the whole thing with it.
